New package: ExperienceReplay v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: ExperienceReplay
- Repository: https://github.com/p-w-rs/ExperienceReplay.jl
- Created by: @p-w-rs
- Version: v0.1.0
- Commit: 6fd40e0581a7d59a389025d94eae3c2cc6d21ae4
- Git reference: HEAD
- Description: An implementation of an experience replay buffer for use in deep rl

goerz commented:
As the bot message says, the problem is in https://github.com/p-w-rs/ExperienceReplay.jl/blob/71622d81c2d0a716e1d1450c66bd70014e56b2ae/Project.toml#L10

It is not allowed to declare compatibility with Julia 2.0 (which will not exist for the foreseeable future, and if it ever does exist, your package is almost certainly not going to be compatible with it)

I would also very strongly recommend that you add some tests before registration.

[noblock]

JuliaHubRegistrator committed:
New package: ExperienceReplay v0.1.0

UUID: 9191f281-19f8-4a32-ad68-bff21db472d5
Repo: https://github.com/p-w-rs/ExperienceReplay.jl.git
Tree: 155974cd6c2d89b2eec7731838caa1d35096a7fc

Registrator tree SHA: 191228b6dd8b9d0e2965ae3e705fe54c51dcfee8
